multiworld/envs/mujoco/sawyer_xyz/sawyer_door.py

```diff
@@ -554,15 +554,58 @@ def __init__(self,
                  min_y_pos=.5,
                  max_y_pos=.7,
                  use_line=True,
+                 num_resets_before_door_reset=1,
+                 num_resets_before_hand_reset=1,
+                 reset_hand_with_door=False,
                  **kwargs
                  ):
         self.quick_init(locals())
         self.max_x_pos = max_x_pos
         self.min_y_pos = min_y_pos
         self.max_y_pos = max_y_pos
         self.use_line = use_line
+        self.door_reset_counter = 0
+        self.hand_reset_counter = 0
+        self.num_resets_before_door_reset = num_resets_before_door_reset
+        self.num_resets_before_hand_reset = num_resets_before_hand_reset
+        self.reset_hand_with_door = reset_hand_with_door
         super().__init__(goal_low=goal_low, goal_high=goal_high, **kwargs)
 
+    def reset(self):
+        if self.reset_hand_with_door:
```

Comment:
We can just get the same results by makign sure `num_resets_before_door_reset` is a multiple of `num_resets_before_hand_reset`.  We might want to do that instead of having this extra hyperparameter